src/parameters.toml

```diff
@@ -1599,6 +1604,26 @@ value = 6
 type = "float"
 description = "Limiter for shape parameter mu for ice in P3 scheme. See eq 3 in Morrison and Milbrandt 2015. Units: [-]"
 
+[water_density]
+value = 1000
+type = "float"
+description = "Density of water in kg/m3"
+
+[freeze_temp]
```

Comment:
We already have that one defined here:

https://github.com/CliMA/ClimaParams.jl/blob/0b7edb933ab81eba907cf06ee713ea463e3db6a4/src/parameters.toml#L1939